components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -207,7 +207,20 @@ def search_and_schedule_new_tasks(
 
         input_type = input_config.type
         if input_type == InputType.FS.value:
-            _process_fs_input_paths(input_config, paths_to_compress_buffer)
+            try:
+                _process_fs_input_paths(input_config, paths_to_compress_buffer)
+            except Exception as err:
+                logger.exception("Failed to process one or more file input paths.")
+                update_compression_job_metadata(
+                    db_cursor,
+                    job_id,
+                    {
+                        "status": CompressionJobStatus.FAILED,
+                        "status_msg": f"Failed to process input: {str(err)}",
```

Comment:
Likely you don't need str() for err, since f string will do the conversion for you. I could be wrong though. can you try removing str() and see how things go?